module/VuFind/src/VuFind/Auth/MultiILS.php

```diff
@@ -123,4 +124,24 @@ public function setCatalog(\VuFind\ILS\Connection $connection)
         }
         parent::setCatalog($connection);
     }
+
+    /**
+     * Test to see if the target ILS supports password Recovery
+     *
+     * @param string $target The ILS we are checking
+     *
+     * @return bool
+     *
+     * @throws \Exception
+     */
+    public function supportsPasswordRecovery($target = null)
+    {
+        $catalog = $this->getCatalog();
+        $driver = $catalog->getDriver();
+        $targetDriver = $driver;
+        if ($target != null) {
+            $targetDriver = $driver->getDriverFromTarget($target);
```

Comment:
It's better to leave the details inside MultiBackend and just pass enough information to it for it to be able to choose the correct driver. Something like:

```
    public function ilsSupportsPasswordRecovery($target)
    {
        $catalog = $this->getCatalog();
        $recoveryConfig = $catalog->checkFunction(
            'recoverPassword',
            ['cat_username' => "$target.123"]
        );
        return $recoveryConfig ? true : false;
    }

```
Admittedly not exactly beautiful, and it's my intention make it possible to just indicate the target more elegantly. Somewhere in future.